improver/calibration/utilities.py

```diff
@@ -355,11 +355,7 @@ def check_forecast_consistency(forecasts: Cube) -> None:
         raise ValueError(msg.format(forecasts.coord("forecast_period").points))
 
 
-def reshape_forecast_predictors(
-    forecast_predictors: CubeList,
-    constr: Optional[iris.Constraint] = None,
-    func: Optional[Callable] = lambda x: x,
-) -> List[ndarray]:
+def reshape_forecast_predictors(forecast_predictors: CubeList) -> List[ndarray]:
```

Comment:
This should probably be renamed as now it only broadcasts predictors without time coords to match those with time coordinates. The docstring will need updating too,

Reply:
I've renamed this.